src/boost_histogram/_internal/hist.py

```diff
@@ -345,6 +345,9 @@ def view(
         """
         return _to_view(self._hist.view(flow))
 
+    def storageType(self) -> str:
+        return str(self._storage_type).rsplit(".", maxsplit=1)[-1][:-2]
```

Comment:
```suggestion
    _storage_type = storage_type
```

And rename `_storage_type` to `storage_type` (change elsewhere too).